### DISCUSSION TITLE
绳斤们

### DISCUSSION THREAD

**Discussion by xiangxliuying** [share121/inter-knot#561] [其他]
发涩图好像会被删，别问我怎么知道的
![Screenshot_2024-08-09-09-15-50-51_149003a2d400f6adb210d7e357a3a646](https://github.com/user-attachments/assets/ee0bba07-fbf3-4c67-9352-fadf46046fd4)

**Comment by xiangxliuying** [comment]
已老实

**Comment by capacitor1** [comment]
> Reply::已老实

+1（悲，看来只能吃独食了...
